depth_image_topic: "kinect/depth/image_raw"
camera_info_topic: "kinect/depth/camera_info"
grasp_frame: "grasp"
world_frame: "world"
debug_output_dir: "/home/philipp/tmp"
generate_debug_ouput: true
show_visualization: true
roi_x: 0
roi_y: 0
roi_width: 640
roi_height: 480
min_depth_tolerance: 0.1
blur_ksize: 15
differentiator: "laplacian"
sobel_ksize: 3
laplacian_ksize: 3
nms_threshold: 10.0
seat_plate_horizontal_position: 0.3